tests/js/jest.config.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
 		'^@woocommerce/blocks-registry$':
 			'<rootDir>/tests/js/woocommerce-blocks-registry',
 		'^uuid$': require.resolve( 'uuid' ),
+		'^jest-utils(.*)$': '<rootDir>/tests/js/utilities$1',
```

Comment:
Similar to WooPay: https://github.com/Automattic/woopay/blob/8962c73aa355f52511b5f87cf04caa7a1233406d/tests/js/jest.config.js#L18